### DISCUSSION TITLE
[gatsby-source-shopify] getting inventory levels for variant in each location

### DISCUSSION THREAD

**Discussion by moreguppy** [gatsbyjs/gatsby#35936] [Help]
Hello — I'm trying to find the stock level for a given variant on all available locations.

I have locations added to the connections option in `gatsby-config.js`:
```
module.exports = {
  {
      resolve: "gatsby-source-shopify",
      options: {
        password: process.env.SHOPIFY_SHOP_PASSWORD,
        storeUrl: process.env.GATSBY_SHOPIFY_STORE_URL,
        shopifyConnections: ["collections", "locations"],
      },
    }
}
```

Then the query looks like: 
```
query MyQuery {
  allShopifyProductVariant {
    nodes {
      inventoryItem {
        inventoryLevels {
          location {
            hasActiveInventory
          }
        }
      }
    }
  }
}
```

But it gets a `"Cannot return null for non-nullable field ShopifyInventoryItem.inventoryLevels."` error message.

In the `gatsby-source-shopify` readme it does mention a bug on legacy ShopifyLocations and I'm just wondering if this is related?

Thanks for your help

**Comment by giovanni-pignataro** [comment]
same too

**Comment by rowanflack** [comment]
Was this ever resolved? That query still produces the same result.
